jupyter_book_to_htmlbook/toc_processing.py

```diff
@@ -59,6 +59,16 @@ def get_book_toc(src_dir: Path) -> list:
 
             if "parts" in toc.keys():
                 compiled_toc.extend(process_parts(toc["parts"], src_dir))
+
+            # check for a preface, and move it ahead of the first part if found
+                preface = [i for i in compiled_toc
+                           if 'preface' in str(i).lower()]
+                # just in case there are multiple matches, confirm only one
+                if len(preface) == 1:
```

Comment:
I'm not sure I understand this logic. I could be wrong, but isn't it valid to have multiple Prefaces?

Reply:
You know what, you're right. Fixed in 21a912c..ab65eae